tests/functions/test_gemini.py

```diff
@@ -129,3 +131,81 @@ def test_generate_videos(self, reset_db: None) -> None:
             assert video_stream['height'] == 720, metadata
             assert video_stream['duration_seconds'] == duration, metadata
             assert audio_stream['duration_seconds'] == duration, metadata
+
+    @pytest.mark.parametrize('default_params', [True, False], ids=['default_params', 'custom_params'])
+    def test_generate_embeddings(self, reset_db: None, default_params: bool) -> None:
+        skip_test_if_not_installed('google.genai')
+        skip_test_if_no_client('gemini')
+        from pixeltable.functions.gemini import generate_embedding
+
+        t = pxt.create_table('test', {'rowid': pxt.Int, 'text': pxt.String})
+
+        # Test embeddings as a computed column
+        config = None
+        expected_dim = 1536
+        if not default_params:
```

Comment:
This isn't quite what I was suggesting. Rather than parameterize the test, just add separate columns that exercise both patterns.

`t.add_computed_column(embedding1=generate_embedding(t.text))`
`t.add_computed_column(embedding2=generate_embedding(t.text, config=..., blahblahblah=...))`

The given implementation doesn't address my concern because it still passes `config=None`, and the purpose is to test what happens if we *don't* give a `config` kwarg. (We've had issues in the past with parameters inadvertently being declared as required, and this being overlooked because they were always specified in every test. So it's important to have tests that specifically *omit* optional kwargs.)